packages/material-ui/src/Chip/Chip.js

```diff
@@ -472,9 +472,10 @@ Chip.propTypes = {
   /**
    * If `true`, the chip will appear clickable, and will raise when pressed,
    * even if the onClick prop is not defined.
-   * If false, the chip will not be clickable, even if onClick prop is defined.
+   * If `false`, the chip will not be clickable, even if onClick prop is defined.
```

Comment:
Maybe just reduce ambiguity here. A note at the end is easily missed.
```suggestion
   * If `false`, the chip will not appear clickable, even if onClick prop is defined.
```

Reply:
I would leave the note at the end for clarity. But in the current form we're directly contradicting a prior statement: "it is clickable [...] it is not clickable. it just looks that way".